charts/sck-otel/templates/_pod.tpl

```diff
@@ -31,18 +31,49 @@ containers:
       {{- end }}
       {{- end }}
     env:
-      - name: MY_POD_IP
+      - name: K8S_NODE_NAME
+        valueFrom:
+          fieldRef:
+            fieldPath: spec.nodeName
+      - name: K8S_NODE_IP
         valueFrom:
           fieldRef:
             apiVersion: v1
-            fieldPath: status.podIP
-      - name: SPLUNK_MEMORY_TOTAL_MIB
-        value: "{{ include "splunk-otel-collector.convertMemToMib" .Values.agent.resources.limits.memory }}"
-      - name: KUBE_NODE_NAME
+            fieldPath: status.hostIP
+      - name: K8S_POD_IP
         valueFrom:
           fieldRef:
             apiVersion: v1
-            fieldPath: spec.nodeName
+            fieldPath: status.podIP
+      - name: K8S_POD_NAME
+        valueFrom:
+          fieldRef:
+            fieldPath: metadata.name
+      - name: K8S_POD_UID
+        valueFrom:
+          fieldRef:
+            fieldPath: metadata.uid
+      - name: K8S_NAMESPACE
+        valueFrom:
+          fieldRef:
+            fieldPath: metadata.namespace
+      {{- if .Values.metricsEnabled }}
```

Comment:
We don't have this values. Should be `splunk-otel-collector.collectMetric` I believe